RELEASE-NOTES.md

```diff
@@ -1,5 +1,32 @@
 # Prefect Release Notes
 
+## Release 2.7.10
+
+### Enhancements
+- Add [`is_schedule_active` option](https://docs.prefect.io/api-ref/prefect/deployments/#prefect.deployments.Deployment) to `Deployment` class to allow control of automatic scheduling — https://github.com/PrefectHQ/prefect/pull/7430
+- Add documentation links to blocks in UI — https://github.com/PrefectHQ/prefect/pull/8210
+- Add Kubernetes kube-system permissions to Prefect agent template for retrieving UUID from kube-system namespace — https://github.com/PrefectHQ/prefect/pull/8205
+- Enhance security of secrets to blocks in UI by incorporating secret fields in blocks' schema — https://github.com/PrefectHQ/prefect/pull/8246
```

Comment:
What does this actually do? Sounds like this does something like "Add support for obfuscation of secrets in nested block fields"